kaolin/conversions/voxelgridconversions.py

```diff
@@ -41,7 +41,7 @@
 import torch.nn.functional as F
 import numpy as np
 import kaolin as kal
-import trimesh
+from kaolin import mcubes
```

Comment:
Is swapping to this cython version helping for the training ? Because this is changing also a lot of other examples.